src/plugins/explore/public/components/query_panel/results_summary/results_summary.scss

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+@import "../shared";
+
+.exploreResultsSummary {
+  @include exploreQueryPanelButton;
+  @include exploreQueryPanelVerticalSeparator;
+}
+
+.exploreResultsSummary_popover_tooltip {
```

Comment:
nitpick: why not `&_popover_tooltip {...}` and `__summaryContent {...}` inside `.exploreResultsSummary { ... }? This way if the name of class change its just one change.


And just fyi, the convention i've been seeing followed some parts of the code, as well as what i'm been following, is this: https://getbem.com/